lib/perl/Genome/InstrumentData/Command/Import/Basic.pm

```diff
@@ -160,26 +161,23 @@ sub _resolve_original_format {
 sub _resolve_instrument_data_properties {
     my $self = shift;
 
-    my $properties = {};
-    if ( $self->instrument_data_properties ) {
-        my $helpers = Genome::InstrumentData::Command::Import::WorkFlow::Helpers->get;
-        $properties = $helpers->key_value_pairs_to_hash( $self->instrument_data_properties );
-        return if not $properties;
+    my $class = 'Genome::InstrumentData::Command::Import::WorkFlow::ResolveInstDataProperties';
+    if ( $self->import_source_name =~ /^CgHub$/i ) {
```

Comment:
Yes. I am working on documenting the whole process. Getting these properties from a DL'd metadata file is from the TCGA importer. I hope to have a better system of creating work flows for importing. For now, the TCGA importer needs to go away. It allows importing of bams that are unusable by the ref align pipeline.